### DISCUSSION TITLE
generator 探讨

### DISCUSSION THREAD

**Discussion by mengshang918** [jd-opensource/drip-form#18] [Ideas]
希望generator有哪些功能？对generator架构有优化？欢迎留下你的看法

**Comment by pk111and222** [comment]
整个生成器还需要增加灵活性
一是整个界面的按钮和功能块有必要的API做显隐的定制。
二是左侧的面板需要可以灵活的定制组件的配置能力。可能有需要完全不依赖主题的组件系统，不能将主题组件作为完全必选项。
三是右侧的属性面板可以灵活的定制属性的编辑和编排能力，目前仅仅支持增加部分功能和属性。而且这个面板也需要支持自定义属性的能力。

**Reply by mengshang918** [reply]
即将开发～

**Reply by mengshang918** [reply]
#39 支持自定义header、自定义组件区域、自定义属性配置区域

**Comment by jiangtao89** [comment]
编辑器区内置组件的属性配置和校验配置支持定制和新增
